src/libstd/collections/mod.rs

```diff
@@ -436,6 +436,8 @@ pub use self::hash_map::HashMap;
 pub use self::hash_set::HashSet;
 
 #[stable(feature = "rust1", since = "1.0.0")]
+#[rustc_deprecated(reason = "moved to std::ops", since = "1.19.0")]
+#[allow(deprecated)]
 pub use alloc::range;
```

Comment:
I might be wrong but I think this reexport will have to be removed otherwise `std::collections::range::RangeArgument` will be usable in stable which I presume isn't intended.